doc/manuals/admin/build_source.md

```diff
@@ -131,6 +133,18 @@ You can generate the RPM for the source code (optional):
 
 * The generated RPMs are placed in directory `~/rpmbuild/RPMS/x86_64`.
 
+### Building MongoDB driver with SASL and SSL support
+
+The procedure is as follows:
+
+```
+wget https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz
+tar xfvz legacy-1.1.2.tar.gz cd mongo-cxx-driver-legacy-1.1.2
+yum install cyrus-sasl-devel
+scons --use-sasl-client --ssl                                   # The build/linux2/normal/libmongoclient.a library is generated as outcome
```

Comment:
we can put comments on a different line

Reply:
For the sake of homogenity I prefer keep using the same style than in other parts of this .md, although in this case it may casue a bit long line. NTC.